polaris.shopify.com/content/getting-started/Develop/index.md

```diff
@@ -0,0 +1,37 @@
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| </li><ul><li>[Component GitHub Package Repo](https://github.com/Shopify/polaris/tree/main/polaris-react)</li><ul><li>[Token GitHub Package Repo](https://github.com/Shopify/polaris/tree/main/polaris-tokens) | </li><ul><li>[Components](https://polaris.shopify.com/components)</li><ul><li>[Tokens](https://polaris.shopify.com/tokens/colors)</li><ul><li>[Icons](https://polaris.shopify.com/icons) | Polaris for VSCode offers automatic autocompletion for Polaris tokens, right inside your favorite code editor.</li><ul><li>[Download the VSCode extension](https://marketplace.visualstudio.com/items?itemName=Shopify.polaris-for-vscode) | Polaris uses [Changsets](https://github.com/changesets/changesets) to handle releasing the npm packages in repository. Our [GitHub action](https://github.com/changesets/action) creates a `version` PR called **"Version Packages"**, then keeps it up to date, recreating it when merged. This PR always has an up-to-date run of `changeset version` and performs a releases when changes are merged to the `main` branch.</li><ul><li>[Perform a release](https://github.com/Shopify/polaris/blob/main/documentation/Releasing.md) |
```

Comment:
Not a blocker to shipping, but I think we could revisit how we display this information. The table makes the content kinda wonky, and when some columns have bullets but no preceding text, it's a little confusing how to read the table.

Reply:
That's a good observation. It seemed like the better option to a resources section with a lot of long form content that had small descriptions and a lot of links. Great idea to revisit.